readme.md

```diff
@@ -17,25 +22,37 @@ answer goes here
 Describe the homework submission process.
 
 ```
-answer goes here
-
+Step 1: Fork the project from a GA repo.
+Step 2: Clone the repo from your own github account to your local machine.
+Step 3: Complete the homework, commit, and push to your github repo.
+Step 4: Create a pull request from your github repo to the GA repo.
 ```
 
 ## Question 3
 
 What is the difference between a fork and a clone?
 
 ```
-answer goes here
+Cloning a repository allows you to make a copy of the repository.
 
+A fork creates a repository in your github profile from the git repo being forked.
+Forking a repository is a more streamlined way to eventually pull from your repo
+into the original repository.
 ```
 
```

Comment:
Again essentially correct, but with 1 micro-quibble: I would say that cloning makes a local copy of a remote repo.